sql-schema/250.sql

```diff
@@ -1,6 +1,6 @@
 INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_client_connections',  'network',  'HTTP Client Connections',  0);
 INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_server_connections',  'network',  'HTTP Server Connections',  0);
 INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_client_connections_active',  'network',  'HTTP Client Connections Active',  0);
-INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'sgos_server_connections_active',  'network',  'HTTP Server Connections Acitve',  0);
```

Comment:
You will have to add a new schema file to correct this.

Reply:
@murrant OK, will get that sorted. Thanks.